rust/src/cfd.rs

```diff
@@ -67,6 +70,7 @@ pub async fn open(order: &Order) -> Result<()> {
     let margin_taker_as_btc = order.margin_taker().0;
     // Convert to msats
     let margin_taker = (margin_taker_as_btc * 100_000_000.0 * 1000.0) as u64;
+    // TODO: is this correct? why do we multiply the leverage for the maker margin?
```

Comment:
The margin is depending on the contract value in BTC, i.e. `contract_value=quantity/price`.
To ensure the maker puts _enough_ margin for the taker to make a profit, we said his margin needs to be equal the `contract_value`, which ends up saying be the `leverage=1`, because, `margin = quantity / (open_price * leverage);` 

For the taker, he can chose a leverage for his margin, meaning, he only has to put up a fraction of the `contract_value`. If we turn the formulas around a bit we get: 
`contract_value=quantity/price` ->
`margin_taker=contract_value/leverage` -->
`contract_value=margin_taker*leverage` --> `margin_maker`.